sn_node_manager/src/bin/cli/main.rs

```diff
@@ -557,6 +565,13 @@ pub enum LocalSubCmd {
         /// The version and path arguments are mutually exclusive.
         #[clap(long)]
         node_version: Option<String>,
+        /// Specify the logging format.
```

Comment:
Hey Josh, sorry for the pedantic issue, but would you mind please specifying the arguments in alphabetical order? There are so many of them and I think it's the best way to keep them organised for easy reference.